content/kubermatic/main/architecture/requirements/cluster-requirements/_index.en.md

```diff
@@ -50,27 +50,29 @@ If you have more than one network adapter, and your Kubernetes components are no
-| TCP      | Inbound   | 10250       | kubelet API           |
-| TCP      | Inbound   | 10255       | Read-only kubelet API |
-| TCP      | Inbound   | 30000-32767 | NodePort Services**   |
-
-** Default port range for [NodePort Services](https://kubernetes.io/docs/concepts/services-networking/service/).
+Please ensure that all nodes in a user cluster can communicate without restriction to ensure functionality of CNI/CSI and Kubernetes itself.
+In addition, user cluster nodes must be able to connect to the Seed cluster's nodeport-proxy. This depends on the [expose strategy]({{< ref "../../../tutorials-howtos/networking/expose-strategies" >}}.
+
+It is recommended to make yourself familiar with the [concept of networking]({{< ref "../../../architecture/concept/kkp-concepts/networking" >}}) in KKP.
+
+| Source                  | Destination                   | Expose Strategy | Ports             | Purpose                                              |
+|-------------------------|-------------------------------|-----------------|-------------------|------------------------------------------------------|
+| KKP Users               | Master Ingress Controller     | Any             | 443*              | Access to KKP Dashboard                              |
+| KKP Operator            | Master cluster Kubernetes API | Any             | 6443*             | Operator access                                      |
+| KKP Operator            | Seed cluster Kubernetes API   | Any             | 6443*             | Operator access                                      |
+| Kubermatic API          | Seed cluster Kubernetes API   | Any             | 6443*             | Operator access                                      |
+| Kubermatic API          | Seed cluster nodeport-proxy   | Tunneling       | 6443              | Access to User Cluster API Endpoints                 | 
+| Kubermatic API          | Seed cluster nodeport-proxy   | NodePort        | 30000-32767**     | Access to User Cluster API Endpoints                 |
+| Kubermatic API          | Seed cluster nodeport-proxy   | LoadBalancer    | 30000-32767**     | Access to User Cluster API Endpoints                 |
+| Seed controller manager | Seed cluster Kubernetes API   | Any             | 6443*             | Controller access                                    |
+| Seed controller manager | Cloud Prorivder API           | Any             | provider specific | Cloud provider api access                            |
+| User cluster nodes      | Seed cluster nodeport-proxy   | Tunneling       | 6443, 8088        | Access to User Cluster API Endpoints and Konnecitivy | 
+| User cluster nodes      | Seed cluster nodeport-proxy   | NodePort        | 30000-32767**     | Access to User Cluster API Endpoints and Konnecitivy |
+| User cluster nodes      | Seed cluster node port-proxy  | LoadBalancer    | 30000-32767**     | Access to User Cluster API Endpoints and Konnecitivy |
```

Comment:
> Destination: Seed cluster node port-proxy 

Is this actually true with the `LoadBalancer` strategy? Shouldn't it say something like "Master cluster Load Balancer IP range"?

Reply:
You are actually right, I think the service is still called nodeport-proxy but its deployed to the user cluster namespace

Reply:
Since we only care about ports here, this information is actually not needed and explained in detail in the concept page.